cli/src/install.sh

```diff
@@ -23,6 +23,11 @@ banner() {
     printf "\n    %bphylum-cli%b installer\n\n" "${GREEN}" "${NC}"
 }
 
+cross_sed() {
+    sed -i'.bak' "$1" "$2"
```

Comment:
I'm curious what happens if someone already has a `.bashrc.bak` file for some other reason. Does this command fail? Does it get overwritten?

Reply:
It doesn't fail, but yes, it does get overwritten.

Reply:
It does not fail...and I _think_ it gets overwritten. The local tests I did for this incantation were done sequentially, before `rm -f`'ing the file. That all worked as well...but I can understand why removing each file directly would be advantageous. I can also see where you might be going with this...to check for the existence of a `*.bak` file _before_ making a new one or at least before deleting it unilaterally.

Reply:
Nah. Just curious. This is probably fine

Reply:
I mean, saying this makes me feel like a bad person 😆 but an easy mitigation would be an odd extension, something  like `sed -i'.phylum.bak' ; rm -f ${2}.phylum.bak`.